js/education.js

```diff
@@ -0,0 +1,648 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
we have JQuery please remove this, please add the third party JS for third CDN's not directly to local.